fallback.c

```diff
@@ -4,6 +4,7 @@
  * Copyright Peter Jones <pjones@redhat.com>
  */
 #include "shim.h"
+#include "utils.h"
```

Comment:
If you move the `shim.h` change from the second commit into this one, this becomes unnecessary.